src/NetBox.cs

```diff
@@ -291,7 +291,7 @@ public static string UrlEncode(string value)
             return null;
 
          byte[] bytes = Encoding.UTF8.GetBytes(value);
-         return Encoding.UTF8.GetString(UrlEncode(bytes, 0, bytes.Length, false /* alwaysCreateNewReturnValue */));
```

Comment:
I think this is not fixing the issue, just disables encoding library wise everywhere.

Reply:
i have moved this code to master, it had to be fixed in one place only.